packages/app-project/components/ClassifyPage/components/ClassifierWrapper/ClassifierWrapperContainer.js

```diff
@@ -42,7 +43,15 @@ export default class ClassifierWrapperContainer extends Component {
       )
     }
 
-    if (project.loadingState === 'success') {
+    if (user.pending) {
```

Comment:
[Major][Bug] `user` here is undefined, I think we need to specify it in...
```
  render () {
    const { authClient, project } = this.props  // ...here
```

Reply:
D'oh! I really should set up linting for this repo, to catch errors like this.

Reply:
Rebased to latest master and fixed. You'll probably need to discard your local branch and pull this one down again.